opal-ui/src/components/BookmarkIcon.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <q-icon
+    :name="name"
+    :color="color"
+    :title="$t(title)"
+    size="sm"
+    class="on-right"
+    @click="authStore.toggleBookmark(props.resource)"
+  />
+</template>
+
+
+<script lang="ts">
```

Comment:
Do not think you need to do this.

Reply:
yes otherwise ` export default` does not work with the "setup" attribute